topics/persistence.md

```diff
@@ -116,7 +124,7 @@ Redis it will re-play the AOF to rebuild the state.
 ### Log rewriting
```

Comment:
maybe all the test you've put in the disadvantages section that describes the rewrite in 7.0 should be moved here?

Reply:
that's also the place to mention preamble if we want

Reply:
OK